source/containers/Form/hooks/textReplacement.ts

```diff
@@ -12,11 +12,29 @@ const replacementRules = [
   ['#lastName', 'user.lastName'],
   ['#datum1', 'date.nextMonth.first'],
   ['#datum2', 'date.nextMonth.last'],
+  ['#månad-1', 'date.previousMonth.currentMonth-1'],
+  ['#månad-2', 'date.previousMonth.currentMonth-2'],
   ['#year', 'date.currentYear'], // this is the current year of next month
```

Comment:
Not really a comment on this PR, but maybe a thing to change... we should stick to either English or Swedish in these hashtags, it feels bad to have #year next to #månad-1.

Reply:
Concur, we should in that case change both #månad, #datum and inform Maria / design.